README.md

```diff
@@ -142,8 +142,9 @@ You can deploy this project adding new lines of code to index.html and styles.cs
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235841880-74134fd0-2ef5-479f-a55b-19303065dce2.png)

**[OPTIONAL]**

- [ ] I think that this is not your Capstone project 😄 Kindly add the correct project name to make your readme descriptive and professional 👍

Reply:
- [OPTIONAL] Please remove this link to your `HTML/CSS & JavaScript capstone project - Conference page`

![image](https://user-images.githubusercontent.com/79058364/235871006-1848c447-811a-4571-95e5-b02d25052f23.png)